src/OrchardCore/OrchardCore.FileStorage.AzureBlob/BlobStorageOptions.cs

```diff
@@ -7,6 +9,13 @@ public abstract class BlobStorageOptions
         /// </summary>
         public string ConnectionString { get; set; }
 
+        /// <summary>
+        /// The reference to a named AzureClient.
```

Comment:
Simplify the comments here by writing something like this
/// The Azure Blob service name.